### ISSUE TITLE
Locale selector: arrow icons should be gray100 not blue

### ISSUE THREAD

**Issue by larahanlon2** [carbon-design-system/ibm-dotcom-library#1979]
## Detailed description

**Describe in detail the issue you're having.**
The icons in the locale modal region cards should be gray100, not blue. 

As-is in Storybook:
<img width="798" alt="Screen Shot 2020-04-06 at 3 55 36 PM" src="https://user-images.githubusercontent.com/53093550/78599507-45e24f00-781f-11ea-81e8-c711c8a2483a.png">

**Is this a feature request (new component, new icon), a bug, or a general issue?**
Design defect

**Is this issue related to a specific component?**
Card

**What did you expect to happen? What happened instead? What would you like to see changed?**
The arrows should be gray100.

As per design spec:
<img width="693" alt="Screen Shot 2020-04-06 at 3 58 22 PM" src="https://user-images.githubusercontent.com/53093550/78599639-7e822880-781f-11ea-9f0e-c6680f36862e.png">

## Steps to reproduce the issue

Open locale selector in Storybook: https://ibmdotcom-react.mybluemix.net/?path=/story/components-locale-modal--default

## Additional information

I think there used to be an override to achieve this but it seems to be missing now? The card component that the locale modal uses has a blue arrow by default – which is correct. However, locale cards should be gray 100.

**Comment by RobertaJHahn** [comment]
Wonil:  Design decision was made to use the blue arrow on the light theme and white on the dark theme.  Wonil will confirm.  Lara will close if not a bug.

**Comment by larahanlon2** [comment]
@wonilsuhibm @RobertaJHahn 

The decision for blue arrows in cards was made in response to **layout patterns** by web simplification. During QA, we noticed that some card patterns had blue arrows while some had gray 100, which was inconsistent when placed on a page. Therefore we decided that all card layout patterns would have blue arrows.

However, the arrows should not be blue in `locale selector` **component**. Since specs they have always been gray100.
